colorcoder/ColorPair.java

```diff
@@ -0,0 +1,47 @@
+package colorcoder;
+
+public class ColorPair {

```

Comment:
Ideally ColorPair should only contain information about the pair. for example major and minor color.
But here conversion of a pair number to a color pair and vice versa is also included. It would be a good idea to segregate those methods into different class.